docs/01GettingStarted/02-Biosensing-Setups/04-ExG-Setup.md

```diff
@@ -88,8 +87,8 @@ Here we will show how to connect to a 3-channel EEG with EMG and ECG on the same
 You can later increase the number of EEG channels based on your needs as long as there enough signal (analog)
 channels (the data channels NxP) on your board.
 
-We will connect the EEG to Cyton as shown in the following image. The two ear clips goes in **SRB** and **BIAS** bottom pins. And the
-three electrode cable connects to the bottom pins of **N1P, N2P, and N3P**.
+We will connect the EEG to Cyton, as shown in the following image. The 2 ear clips go in **SRB** and **BIAS** bottom pins. The
+3 electrode cable connects to the bottom pins of **N1P, N2P, and N3P**.
```

Comment:
Write out the numbers here as they are less than ten. 2 -> two . 

also
~~The 3 electrode cable connects to the bottom pins of **N1P, N2P, and N3P**.~~
 The three electrode cables connect